src/imports/book.rs

```diff
@@ -161,6 +161,7 @@ impl ParseBook {
     let adts = std::mem::take(&mut self.adts);
     let mut new_adts = IndexMap::new();
     let mut ctrs_map = IndexMap::new();
+    let mut new_ctrs = IndexMap::new();
```

Comment:
This is for https://github.com/HigherOrderCO/Bend/issues/674 right?

Can you add a test?